osu.ElasticIndexer/UnrunnableProcessor.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Threading;
+using osu.Server.QueueProcessor;
+
+namespace osu.ElasticIndexer
+{
+    public class UnrunnableProcessor : QueueProcessor<ScoreItem>
+    {
+        private static readonly string queue_name = $"score-index-{AppSettings.Schema}";
+
+        public string QueueName { get; }
+
+        internal UnrunnableProcessor()
+            : base(new QueueConfiguration { InputQueueName = queue_name })
+        {
+            QueueName = queue_name;
+        }
+
+        public new void Run(CancellationToken cancellation = default)
```

Comment:
This doesn't really add much safety as if the local variable is of the wrong type you can still call the `base` method.